<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>421</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Capitalisation:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>359</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tax rate: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="depositInteterstRateTextEdit">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>328</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>297</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Term: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>260</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Loan ammount: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="depositTermTextEdit">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>297</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loanAnnButton">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>450</y>
      <width>118</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Loan (Annuity)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>390</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Payments periodicity: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="depositAmmoutTextEdit">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>260</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>320</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Interest rate: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>290</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Term: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="loanAmmountTextEdit">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>260</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interestRateTextEdit">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>320</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="termTextEdit">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>290</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="loanOutput">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>488</y>
      <width>289</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="expressionTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>14</y>
      <width>597</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loanDifButton">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>450</y>
      <width>134</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Loan (Diffirential)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xTextEdit">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>190</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Expression: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="depositButton">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>450</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>91</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X value: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Result: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="resultTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>597</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="taxRateTextEdit">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>359</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="depositOutout">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>498</y>
      <width>294</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>328</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Interest rate: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>260</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit ammount: </string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CapitalisationCheckBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>423</y>
      <width>19</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="paymentsPeriodicity">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>390</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Plot" native="true">
    <property name="geometry">
     <rect>
      <x>869</x>
      <y>14</y>
      <width>451</width>
      <height>541</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>600</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Xmin:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>630</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Xmax:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_15">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>600</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ymin:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>630</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ymax:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Xmin">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>600</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-100000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Xmax">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>630</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ymin">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>600</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-1000000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ymax">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>630</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_17">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>660</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y auto:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="Yauto">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>660</y>
      <width>19</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="polishNotationTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>597</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Polish notation:</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="actioncalculateButtonPressed">
   <property name="text">
    <string>calculateButtonPressed</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
